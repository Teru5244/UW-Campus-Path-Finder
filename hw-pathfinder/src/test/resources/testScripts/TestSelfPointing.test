# Test the searching algorithm on a self-pointing node

CreateGraph selfPointing

AddNode selfPointing A

FindPath selfPointing A A