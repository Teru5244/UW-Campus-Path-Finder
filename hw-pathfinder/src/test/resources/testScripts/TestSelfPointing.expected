# Test the searching algorithm on a self-pointing node

created graph selfPointing

added node A to selfPointing

path from A to A:
A to A with weight 0.000
total cost: 0.000